discovery/scaleway/instance.go

```diff
@@ -34,17 +34,28 @@ import (
 const (
 	instanceLabelPrefix = metaLabelPrefix + "instance_"
 
-	instanceIDLabel        = instanceLabelPrefix + "id"
-	instancePrivateIPv4    = instanceLabelPrefix + "private_ipv4"
-	instancePublicIPv4     = instanceLabelPrefix + "public_ipv4"
-	instancePublicIPv6     = instanceLabelPrefix + "public_ipv6"
-	instanceImageNameLabel = instanceLabelPrefix + "image_name"
-	instanceNameLabel      = instanceLabelPrefix + "name"
-	instanceProjectLabel   = instanceLabelPrefix + "project_id"
-	instanceStateLabel     = instanceLabelPrefix + "status"
-	instanceTagsLabel      = instanceLabelPrefix + "tags"
-	instanceTypeLabel      = instanceLabelPrefix + "type"
-	instanceZoneLabel      = instanceLabelPrefix + "zone"
+	instanceBootTypeLabel          = instanceLabelPrefix + "boot_type"
+	instanceHostnameLabel          = instanceLabelPrefix + "hostname"
+	instanceIDLabel                = instanceLabelPrefix + "id"
+	instanceImageArchLabel         = instanceLabelPrefix + "image_arch"
+	instanceImageIDLabel           = instanceLabelPrefix + "image_id"
+	instanceImageNameLabel         = instanceLabelPrefix + "image_name"
+	instanceLocationClusterID      = instanceLabelPrefix + "location_cluster_id"
+	instanceLocationHypervisorID   = instanceLabelPrefix + "location_hypervisor_id"
+	instanceLocationNodeID         = instanceLabelPrefix + "location_node_id"
+	instanceNameLabel              = instanceLabelPrefix + "name"
+	instanceOrganizationLabel      = instanceLabelPrefix + "organization"
```

Comment:
For consistency it should be `organization_id`

```suggestion
	instanceOrganizationLabel      = instanceLabelPrefix + "organization_id"
```

Reply:
I agree, I did pick organization because Prometheus should 'blindly' dump the API and the API is using organization, not organization_id. But it will probably be less confusing to use organization_id.